### PR TITLE
Metrics webapp setup

### PR DESCRIPTION
**Pull request by Yerimah**
In this branch we;

- Selected the financial Modelling API.
- Built files structure for the React app.
- Prepared routes and navigation in the app
- Called data from the Api
- Created test for the application
- Styled the components
- Fixed Linter errors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,91 @@
-# stock-metrics
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Stock Metrics App
+
+![Stocks app](https://user-images.githubusercontent.com/71140133/158986383-0d137bee-3297-4d46-9ee2-0f68a9ef9194.PNG)
+
+
+- [Live Demo](https://stockmetricapp.netlify.app/details)
```
Comment (Whoistolu) [inline README.md]: - [x] [REQUIRED] Great job deploying this project using Netlify, however, the link seems to be broken. Kindly check the log files to fix this or redeploy using Heroku or Github Pages in order to meet up with the requiremnt.
Screenshot from 2022-03-18 18-44-33

File: src/Components/Pages/Details.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+          </li>
+        ))
+
+}
+
+    </section>
+  );
+};
+
+export default Details;
```
Comment (Whoistolu) [inline src/Components/Pages/Details.js:84]: - [ ] [OPTIONAL] Kindly note that on reload/refresh of the detail page, there is no data been displayed. You can fix this using `useParam`.
Screenshot from 2022-03-18 19-54-12

File: src/Components/Pages/Dashboard.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+//     <section>
+//       <Header />
+//       <CardList />
+
+//     </section>
+//   );
+// };
+
+// export default Dashboard;
+
```
Comment (Whoistolu) [inline src/Components/Pages/Dashboard.js]: - [x] [OPTIONAL] Please remove all unused lines of code in order for us to have a clean code-base.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Yerimah 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Yerimah) [reply README.md]: Resolved

File: src/Components/Pages/Dashboard.js (hunk 3 above)
Comment (Yerimah) [reply src/Components/Pages/Dashboard.js]: All unused lines of code have been removed

**Review by er-contreras — APPROVED** [review]
Hi @Yerimah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- Well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**